[dcmtk] Build failure

Package: dcmtk[core,tiff,tools]:arm64-osx -> 3.6.7#5

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 14.0.3.14030022
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 3437b8e9f 2023-09-01 (2 days ago)

**To Reproduce**

`vcpkg x-set-installed dcmtk[core,tiff,tools]`
**Failure logs**

```
-- Using cached DCMTK-dcmtk-a137f1aff4e1df3fbefe53ee8b160973c74c96dd.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/DCMTK-dcmtk-a137f1aff4e1df3fbefe53ee8b160973c74c96dd.tar.gz
-- Applying patch dcmtk.patch
-- Applying patch windows-patch.patch
-- Applying patch fix-pc-format.patch
-- Using source at /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean
-- Found external ninja('1.11.1').
-- Configuring arm64-osx
CMake Warning at /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:318 (message):
  The following variables are not used in CMakeLists.txt:

      INSTALL_HEADERS
      INSTALL_OTHER

  Please recheck them and remove the unnecessary options from the
  `vcpkg_cmake_configure` call.

  If these options should still be passed for whatever reason, please use the
  `MAYBE_UNUSED_VARIABLES` argument.
Call Stack (most recent call first):
  ports/dcmtk/portfile.cmake:25 (vcpkg_cmake_configure)
  scripts/ports.cmake:147 (include)


-- Building arm64-osx-dbg
-- Building arm64-osx-rel
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/tools/cmake-3.27.1-osx/cmake-3.27.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Release --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/arm64-osx-rel
    See logs for more information:
      /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/install-arm64-osx-rel-out.log

Call Stack (most recent call first):
  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/dcmtk/portfile.cmake:44 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/install-arm64-osx-rel-out.log</summary>

```
Change Dir: '/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/arm64-osx-rel'

Run Build Command(s): /opt/homebrew/bin/ninja -v -v -j11 install
[1/1275] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DDCMTK_BUILD_IN_PROGRESS -DUSE_NULL_SAFE_OFSTRING -D_BSD_COMPAT -D_BSD_SOURCE -D_OSF_SOURCE -D_REENTRANT -D_XOPEN_SOURCE_EXTENDED -Dofstd_EXPORTS -I/opt/homebrew/Cellar/openjpeg/2.5.0_1/include/openjpeg-2.5 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/arm64-osx-rel/config/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/ofstd/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/oflog/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmdata/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimgle/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimage/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmnet/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsr/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsign/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmwlm/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmqrdb/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpstat/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmrt/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmiod/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmfg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmseg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtract/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpmap/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmect/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/include -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -std=gnu++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -MD -MT ofstd/libsrc/CMakeFiles/ofstd.dir/oferror.cc.o -MF ofstd/libsrc/CMakeFiles/ofstd.dir/oferror.cc.o.d -o ofstd/libsrc/CMakeFiles/ofstd.dir/oferror.cc.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/ofstd/libsrc/oferror.cc
...
Skipped 1060 lines
...
[408/1275] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDCMTK_BUILD_IN_PROGRESS -DUSE_NULL_SAFE_OFSTRING -D_BSD_COMPAT -D_BSD_SOURCE -D_OSF_SOURCE -D_REENTRANT -D_XOPEN_SOURCE_EXTENDED -Dijg8_EXPORTS -I/opt/homebrew/Cellar/openjpeg/2.5.0_1/include/openjpeg-2.5 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/arm64-osx-rel/config/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/ofstd/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/oflog/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmdata/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimgle/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimage/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmnet/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsr/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsign/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmwlm/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmqrdb/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpstat/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmrt/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmiod/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmfg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmseg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtract/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpmap/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmect/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/include -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -MD -MT dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jfdctfst.c.o -MF dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jfdctfst.c.o.d -o dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jfdctfst.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg8/jfdctfst.c
[409/1275] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDCMTK_BUILD_IN_PROGRESS -DUSE_NULL_SAFE_OFSTRING -D_BSD_COMPAT -D_BSD_SOURCE -D_OSF_SOURCE -D_REENTRANT -D_XOPEN_SOURCE_EXTENDED -Dijg8_EXPORTS -I/opt/homebrew/Cellar/openjpeg/2.5.0_1/include/openjpeg-2.5 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/arm64-osx-rel/config/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/ofstd/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/oflog/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmdata/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimgle/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimage/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmnet/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsr/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsign/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmwlm/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmqrdb/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpstat/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmrt/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmiod/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmfg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmseg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtract/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpmap/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmect/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/include -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -MD -MT dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jidctflt.c.o -MF dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jidctflt.c.o.d -o dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jidctflt.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg8/jidctflt.c
[410/1275] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDCMTK_BUILD_IN_PROGRESS -DUSE_NULL_SAFE_OFSTRING -D_BSD_COMPAT -D_BSD_SOURCE -D_OSF_SOURCE -D_REENTRANT -D_XOPEN_SOURCE_EXTENDED -Dijg8_EXPORTS -I/opt/homebrew/Cellar/openjpeg/2.5.0_1/include/openjpeg-2.5 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/arm64-osx-rel/config/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/ofstd/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/oflog/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmdata/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimgle/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimage/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmnet/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsr/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsign/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmwlm/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmqrdb/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpstat/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmrt/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmiod/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmfg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmseg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtract/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpmap/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmect/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/include -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -MD -MT dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jfdctint.c.o -MF dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jfdctint.c.o.d -o dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jfdctint.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg8/jfdctint.c
[411/1275] : && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  dcmimage/apps/CMakeFiles/dcm2pnm.dir/dcm2pnm.cc.o -o bin/dcm2pnm  lib/libdcmimage.a  lib/libdcmimgle.a  lib/libdcmdata.a  lib/liboflog.a  lib/libofstd.a  -lpthread  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/lib/libtiff.a  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/lib/libjpeg.a && :
FAILED: bin/dcm2pnm 
: && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  dcmimage/apps/CMakeFiles/dcm2pnm.dir/dcm2pnm.cc.o -o bin/dcm2pnm  lib/libdcmimage.a  lib/libdcmimgle.a  lib/libdcmdata.a  lib/liboflog.a  lib/libofstd.a  -lpthread  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/lib/libtiff.a  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/lib/libjpeg.a && :
Undefined symbols for architecture arm64:
  "_deflate", referenced from:
      _PixarLogPostEncode in libtiff.a(tif_pixarlog.c.o)
      _PixarLogEncode in libtiff.a(tif_pixarlog.c.o)
      _ZIPPostEncode in libtiff.a(tif_zip.c.o)
      _ZIPEncode in libtiff.a(tif_zip.c.o)
  "_deflateEnd", referenced from:
      _PixarLogCleanup in libtiff.a(tif_pixarlog.c.o)
      _ZIPSetupDecode in libtiff.a(tif_zip.c.o)
      _ZIPCleanup in libtiff.a(tif_zip.c.o)
  "_deflateInit_", referenced from:
      _PixarLogSetupEncode in libtiff.a(tif_pixarlog.c.o)
      _ZIPSetupEncode in libtiff.a(tif_zip.c.o)
  "_deflateParams", referenced from:
      _PixarLogVSetField in libtiff.a(tif_pixarlog.c.o)
      _ZIPVSetField in libtiff.a(tif_zip.c.o)
  "_deflateReset", referenced from:
      _PixarLogPreEncode in libtiff.a(tif_pixarlog.c.o)
      _ZIPPreEncode in libtiff.a(tif_zip.c.o)
  "_inflate", referenced from:
      _PixarLogDecode in libtiff.a(tif_pixarlog.c.o)
      _ZIPDecode in libtiff.a(tif_zip.c.o)
  "_inflateEnd", referenced from:
      _PixarLogCleanup in libtiff.a(tif_pixarlog.c.o)
      _ZIPSetupEncode in libtiff.a(tif_zip.c.o)
      _ZIPCleanup in libtiff.a(tif_zip.c.o)
  "_inflateInit_", referenced from:
      _PixarLogSetupDecode in libtiff.a(tif_pixarlog.c.o)
      _ZIPSetupDecode in libtiff.a(tif_zip.c.o)
  "_inflateReset", referenced from:
      _PixarLogPreDecode in libtiff.a(tif_pixarlog.c.o)
      _ZIPPreDecode in libtiff.a(tif_zip.c.o)
  "_lzma_code", referenced from:
      _LZMADecode in libtiff.a(tif_lzma.c.o)
      _LZMAPostEncode in libtiff.a(tif_lzma.c.o)
      _LZMAEncode in libtiff.a(tif_lzma.c.o)
  "_lzma_end", referenced from:
      _LZMASetupDecode in libtiff.a(tif_lzma.c.o)
      _LZMASetupEncode in libtiff.a(tif_lzma.c.o)
      _LZMACleanup in libtiff.a(tif_lzma.c.o)
  "_lzma_lzma_preset", referenced from:
      _TIFFInitLZMA in libtiff.a(tif_lzma.c.o)
      _LZMAVSetField in libtiff.a(tif_lzma.c.o)
  "_lzma_memusage", referenced from:
      _LZMADecode in libtiff.a(tif_lzma.c.o)
  "_lzma_stream_decoder", referenced from:
      _LZMAPreDecode in libtiff.a(tif_lzma.c.o)
      _LZMADecode in libtiff.a(tif_lzma.c.o)
  "_lzma_stream_encoder", referenced from:
      _LZMAVSetField in libtiff.a(tif_lzma.c.o)
      _LZMAPreEncode in libtiff.a(tif_lzma.c.o)
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
[412/1275] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDCMTK_BUILD_IN_PROGRESS -DUSE_NULL_SAFE_OFSTRING -D_BSD_COMPAT -D_BSD_SOURCE -D_OSF_SOURCE -D_REENTRANT -D_XOPEN_SOURCE_EXTENDED -Dijg8_EXPORTS -I/opt/homebrew/Cellar/openjpeg/2.5.0_1/include/openjpeg-2.5 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/arm64-osx-rel/config/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/ofstd/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/oflog/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmdata/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimgle/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimage/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmnet/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsr/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsign/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmwlm/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmqrdb/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpstat/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmrt/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmiod/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmfg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmseg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtract/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpmap/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmect/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/include -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -MD -MT dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jidctfst.c.o -MF dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jidctfst.c.o.d -o dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jidctfst.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg8/jidctfst.c
[413/1275] : && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  dcmimage/apps/CMakeFiles/dcmquant.dir/dcmquant.cc.o -o bin/dcmquant  lib/libdcmimage.a  lib/libdcmimgle.a  lib/libdcmdata.a  lib/liboflog.a  lib/libofstd.a  -lpthread  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/lib/libtiff.a  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/lib/libjpeg.a && :
[414/1275] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDCMTK_BUILD_IN_PROGRESS -DUSE_NULL_SAFE_OFSTRING -D_BSD_COMPAT -D_BSD_SOURCE -D_OSF_SOURCE -D_REENTRANT -D_XOPEN_SOURCE_EXTENDED -Dijg8_EXPORTS -I/opt/homebrew/Cellar/openjpeg/2.5.0_1/include/openjpeg-2.5 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/arm64-osx-rel/config/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/ofstd/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/oflog/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmdata/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimgle/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimage/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmnet/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsr/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsign/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmwlm/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmqrdb/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpstat/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmrt/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmiod/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmfg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmseg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtract/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpmap/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmect/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/include -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -MD -MT dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jidctint.c.o -MF dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jidctint.c.o.d -o dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jidctint.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg8/jidctint.c
[415/1275] : && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  dcmimage/apps/CMakeFiles/dcmscale.dir/dcmscale.cc.o -o bin/dcmscale  lib/libdcmimage.a  lib/libdcmimgle.a  lib/libdcmdata.a  lib/liboflog.a  lib/libofstd.a  -lpthread  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/lib/libtiff.a  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/lib/libjpeg.a && :
[416/1275] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDCMTK_BUILD_IN_PROGRESS -DUSE_NULL_SAFE_OFSTRING -D_BSD_COMPAT -D_BSD_SOURCE -D_OSF_SOURCE -D_REENTRANT -D_XOPEN_SOURCE_EXTENDED -Dijg8_EXPORTS -I/opt/homebrew/Cellar/openjpeg/2.5.0_1/include/openjpeg-2.5 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/arm64-osx-rel/config/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/ofstd/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/oflog/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmdata/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimgle/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimage/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmnet/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsr/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsign/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmwlm/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmqrdb/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpstat/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmrt/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmiod/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmfg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmseg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtract/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpmap/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmect/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/include -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -MD -MT dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jidctred.c.o -MF dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jidctred.c.o.d -o dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jidctred.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg8/jidctred.c
[417/1275] : && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names  dcmimage/apps/CMakeFiles/dcmicmp.dir/dcmicmp.cc.o -o bin/dcmicmp  lib/libdcmimage.a  lib/libdcmimgle.a  lib/libdcmdata.a  lib/liboflog.a  lib/libofstd.a  -lpthread  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/lib/libtiff.a  /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/lib/libjpeg.a && :
[418/1275] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DDCMTK_BUILD_IN_PROGRESS -DUSE_NULL_SAFE_OFSTRING -D_BSD_COMPAT -D_BSD_SOURCE -D_OSF_SOURCE -D_REENTRANT -D_XOPEN_SOURCE_EXTENDED -Ddcmjpeg_EXPORTS -I/opt/homebrew/Cellar/openjpeg/2.5.0_1/include/openjpeg-2.5 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/arm64-osx-rel/config/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/ofstd/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/oflog/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmdata/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimgle/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimage/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmnet/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsr/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsign/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmwlm/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmqrdb/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpstat/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmrt/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmiod/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmfg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmseg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtract/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpmap/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmect/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg8 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg12 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg16 -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -std=gnu++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -MD -MT dcmjpeg/libsrc/CMakeFiles/dcmjpeg.dir/djrplol.cc.o -MF dcmjpeg/libsrc/CMakeFiles/dcmjpeg.dir/djrplol.cc.o.d -o dcmjpeg/libsrc/CMakeFiles/dcmjpeg.dir/djrplol.cc.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libsrc/djrplol.cc
[419/1275] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DDCMTK_BUILD_IN_PROGRESS -DUSE_NULL_SAFE_OFSTRING -D_BSD_COMPAT -D_BSD_SOURCE -D_OSF_SOURCE -D_REENTRANT -D_XOPEN_SOURCE_EXTENDED -Dijg8_EXPORTS -I/opt/homebrew/Cellar/openjpeg/2.5.0_1/include/openjpeg-2.5 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/arm64-osx-rel/config/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/ofstd/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/oflog/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmdata/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimgle/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimage/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmnet/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsr/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsign/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmwlm/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmqrdb/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpstat/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmrt/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmiod/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmfg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmseg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtract/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpmap/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmect/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/include -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -MD -MT dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jdmarker.c.o -MF dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jdmarker.c.o.d -o dcmjpeg/libijg8/CMakeFiles/ijg8.dir/jdmarker.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg8/jdmarker.c
[420/1275] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DDCMTK_BUILD_IN_PROGRESS -DUSE_NULL_SAFE_OFSTRING -D_BSD_COMPAT -D_BSD_SOURCE -D_OSF_SOURCE -D_REENTRANT -D_XOPEN_SOURCE_EXTENDED -Ddcmjpeg_EXPORTS -I/opt/homebrew/Cellar/openjpeg/2.5.0_1/include/openjpeg-2.5 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/arm64-osx-rel/config/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/ofstd/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/oflog/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmdata/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimgle/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimage/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmnet/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsr/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsign/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmwlm/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmqrdb/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpstat/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmrt/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmiod/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmfg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmseg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtract/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpmap/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmect/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg8 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg12 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg16 -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -std=gnu++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -MD -MT dcmjpeg/libsrc/CMakeFiles/dcmjpeg.dir/djrploss.cc.o -MF dcmjpeg/libsrc/CMakeFiles/dcmjpeg.dir/djrploss.cc.o.d -o dcmjpeg/libsrc/CMakeFiles/dcmjpeg.dir/djrploss.cc.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libsrc/djrploss.cc
[421/1275] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DDCMTK_BUILD_IN_PROGRESS -DUSE_NULL_SAFE_OFSTRING -D_BSD_COMPAT -D_BSD_SOURCE -D_OSF_SOURCE -D_REENTRANT -D_XOPEN_SOURCE_EXTENDED -Ddcmjpeg_EXPORTS -I/opt/homebrew/Cellar/openjpeg/2.5.0_1/include/openjpeg-2.5 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/arm64-osx-rel/config/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/ofstd/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/oflog/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmdata/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimgle/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmimage/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtls/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmnet/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsr/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmsign/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmwlm/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmqrdb/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpstat/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmrt/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmiod/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmfg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmseg/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmtract/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmpmap/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmect/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/include -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg8 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg12 -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libijg16 -fPIC -D_XOPEN_SOURCE_EXTENDED -D_BSD_SOURCE -D_BSD_COMPAT -D_OSF_SOURCE -D_DARWIN_C_SOURCE -O3 -DNDEBUG -std=gnu++17 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -Wall -MD -MT dcmjpeg/libsrc/CMakeFiles/dcmjpeg.dir/djutils.cc.o -MF dcmjpeg/libsrc/CMakeFiles/dcmjpeg.dir/djutils.cc.o.d -o dcmjpeg/libsrc/CMakeFiles/dcmjpeg.dir/djutils.cc.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/dcmtk/src/73c74c96dd-7e3f7568d3.clean/dcmjpeg/libsrc/djutils.cc
ninja: build stopped: subcommand failed.
```
</details>**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "vcpkg-configuration": {
    "overlay-ports": [
      "overlay-ports"
    ]
  },
  "name": "lichtsteuerung",
  "version-string": "2.0.0",
  "dependencies": [
    {
      "name": "aubio",
      "default-features": false
    },
    {
      "name": "booserwert",
      "default-features": false,
      "features": [
        "er"
      ],
      "platform": "osx"
    },
    {
      "name": "boost",
      "platform": "osx"
    },
    "boost-coroutine",
    "boost-coroutine2",
    "boost-stacktrace",
    "kubazip",
    {
      "name": "libbacktrace",
      "platform": "!(windows & !mingw)"
    },
    "rtaudio",
    "segvcatch",
    "span-lite"
  ],
  "features": {
    "qt": {
      "description": "Uses vcpkg to build vcpkg",
      "dependencies": [
        "qt5compat",
        "qtmqtt",
        "qtnetworkauth",
        "qtquickcontrols2",
        "qtsvg",
        "qtwebsockets"
      ]
    }
  }
}

```
</details>

--- a/ports/dcmtk/portfile.cmake
+++ b/ports/dcmtk/portfile.cmake
@@ -11,6 +11,14 @@ vcpkg_from_github(
         fix_link_tiff.patch
 )
 
+# Prefix all exported API symbols of vendored libjpeg with "dcmtk_"
+file(GLOB src_files "${SOURCE_PATH}/dcmjpeg/libijg*/*.c" "${SOURCE_PATH}/dcmjpeg/libijg*/*.h")
+foreach(file_path ${src_files})
+    file(READ "${file_path}" file_string)
+    string(REGEX REPLACE "(#define[ \t\r\n]+[A-Za-z0-9_]*[ \t\r\n]+)(j[a-z]+[0-9]+_)" "\\1dcmtk_\\2" file_string "${file_string}")
+    file(WRITE "${file_path}" "${file_string}")
+endforeach()
+
 vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
     FEATURES
         "iconv"   DCMTK_WITH_ICONV

--- a/ports/dcmtk/vcpkg.json
+++ b/ports/dcmtk/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "dcmtk",
   "version": "3.6.8",
-  "port-version": 2,
+  "port-version": 3,
   "description": "This DICOM ToolKit (DCMTK) package consists of source code, documentation and installation instructions for a set of software libraries and applications implementing part of the DICOM/MEDICOM Standard.",
   "homepage": "https://github.com/DCMTK/dcmtk",
   "license": "BSD-3-Clause OR BSD-2-Clause OR libtiff OR MIT OR Zlib OR Libpng",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2174,7 +2174,7 @@
     },
     "dcmtk": {
       "baseline": "3.6.8",
-      "port-version": 2
+      "port-version": 3
     },
     "debug-assert": {
       "baseline": "1.3.3",

--- a/versions/d-/dcmtk.json
+++ b/versions/d-/dcmtk.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "43cc4b7f4cafcf246412bd814cc0e3cd6f60db6f",
+      "version": "3.6.8",
+      "port-version": 3
+    },
+    {
       "git-tree": "86607f6cdca69d2aca997eead93dfa2b767679a1",
       "version": "3.6.8",
       "port-version": 2

